cluster_management/src/main/java/com/pinterest/rocksplicator/LeaderFollowerStateModelFactory.java

```diff
@@ -471,6 +478,46 @@ public void onBecomeFollowerFromOffline(Message message, NotificationContext con
           Utils.restoreLocalDBFromS3(adminPort, dbName, s3Bucket, s3Path, snapshotHost,
               snapshotPort);
         }
+
+        try {
+          LOG.error(dbName + " Catching up live updates to leader");
+          Map<String, String> liveHostAndRole = getLiveHostAndRole(context, dbName);
```

Comment:
are we expecting the stateMap changed since the last getExternalView? We are not here, right?
- if so, can we catch the externalView instead of get from zk again.

Reply:
Since backup/restore can take up to an hour (and there no is mutex lock), high chances of external view changing. So we would want to get it again.